tests/DCPS/Compiler/xcdr/xcdr.cpp

```diff
@@ -2533,9 +2548,7 @@ void build_expected_union(DataVec& expected, FieldFilter field_filter, bool keye
   if (include_unkeyed) {
     non_keys = DataView(key_only_union_non_keys_expected_base);
   }
-  if (include_possible_keyed) {
-    serialize_u32(expected, keys.size + non_keys.size);
-  }
+  serialize_u32(expected, keys.size + non_keys.size);
```

Comment:
Is the serialization of unions changing?

Reply:
Oh, this is the keyed delimiter... If we're doing this, at the very least we have to announce in the news that their serialization might break.

Reply:
Yes, it's for the change to KeyOnly serialization of union. I'll add a news item.